config/assembleCoadd.py

```diff
@@ -2,10 +2,10 @@
 from lsst.obs.lsstSim.selectFluxMag0 import SelectLsstSimFluxMag0Task
 from lsst.pipe.tasks.scaleZeroPoint import SpatialScaleZeroPointTask
 
-config.select.retarget(SelectLsstImagesTask)
+#config.select.retarget(SelectLsstImagesTask)
```

Comment:
You are right.  I just got rid of this config files since we no longer use those specialized tasks.